pyof/v0x01/common/phy_port.py

```diff
@@ -188,7 +188,7 @@ class ListOfPhyPorts(basic_types.FixedTypeList):
     Attributes:
         items (optional): Instance or a list of instances of PhyPort
     """
-    def __init__(self, items=[]):
-        basic_types.FixedTypeList.__init__(self,
-                                           pyof_class=PhyPort,
-                                           items=items)
+    def __init__(self, items=None):
+        super().__init__(self,
```

Comment:
As we can see in [PEP 3135](https://www.python.org/dev/peps/pep-3135/) and in [Python 3.5.1 docs](https://docs.python.org/3/library/functions.html#super), `super()` is not deprecated. Is this change still necessary?